_posts/2021-01-23-Agrelon.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+date: 2021-01-23
+title: Agrelon
+author: cgy
+---
+
+{% assign author = site.data.authors[page.author]%}
+{{author.name}}
+
+ORCID: <a href="https://orcid.org/{{ author.orcid }}" title="{{author.name}}">{{author.orcid}}</a>
+
+Choosing an ontology is always a difficult task (see [Triplestores,
+Ontologies, and Reasoning]({% post_url
+2020-06-22-Triplestores-Onotologies-Reasoning %})).  There are many
+different considerations to make.  Among them is: availability,
+suitability for the dataset, and complexity.  There is also the ever
+present pull of writing an ontology for your data from scratch, which
+is often best avoided but is always a temptation.
+
+For IrishGen, I decided early on to use the [Relationship
+ontology](http://purl.org/vocab/relationship/).  This was due to the
+fact that it covered most of what was necessary and any additional
+predicates needed or any adjustments could be incorporated in a
+follow-on ontology.  Moreover, the Relationship ontology was built on
+top of the very well-known and universally used [Foaf
+ontology](http://xmlns.com/foaf/0.1/).
+
+In the course of the development of IrishGen, more predicates needed
+to be added to cover some more areas that the Relationship ontology
+did not.  Additional tweaks were also needed to make the `parentOf`
+relation into a subproperty of the `ancestorOf` predicate (and
+inversely for the `childOf`) as this would not allow transitivity
+backwards (also remember that your parents are your ancestors as
+well).
+
+In private email conversations with other individuals who are
+interested in the Semantic Web and Linked Data, a serious flaw in the
+way in which the Relationship ontology uses `owl:equivalentClass` was
+brought to my attention.  They proposed that IrishGen use the ontology
```

Comment:
"the ontology" > "an ontology"? Or is it really the only one?

Reply:
You are right.  I will amend slightly and send it to you.